### PR TITLE
Navbar auth

### PR DESCRIPTION
**Pull request by dhuber666**

### CODE REVIEW

**Comment by dhuber666** [comment]
I have a question @charliearlie 

Let's take the NavBar Component. It uses proptypes like this:

`NavBar.propTypes = {
	clickerCount: PropTypes.number.isRequired,
	user: PropTypes.object.isRequired,
};`

The first time it renders it doesn't have a clickerCount or user since these are async functions in the App Component. 

So it always complains that it's null. It's just a warning but I hate warnings too :D 
Do you know to get rid of it? I googled now for over an hour but can't find any solution. 

I could set a default prop but then eslint complains that it is marked as required. It really is required or it won't work.

Thx for your help!

**Comment by dhuber666** [comment]
Do you know how to deploy the newest master (everytime we merge into master) to heroku?

**Comment by charliearlie** [comment]
In regards to the navbar, we'll just make them not required for now. It's hacky but it'll remove the errors

File: client/.eslintrc [hunk 1]
```diff
@@ -10,7 +10,8 @@
 		"jsx-a11y/anchor-is-valid": 0,
 		"react/no-did-mount-set-state": 0,
 		"linebreak-style": 0,
-		"class-methods-use-this": 0
+		"class-methods-use-this": 0,
+		"react/forbid-prop-types": 0
```
Comment (charliearlie) [inline client/.eslintrc]: Again I must ask, as I have to with any eslint change.

**Comment by dhuber666** [comment]
I already fixed that

**Pull request merged**